src/cg_ftoc.c

```diff
@@ -188,7 +188,7 @@ CGNSDLL void FMNAME(cg_configure_f, CG_CONFIGURE_F) (cgint_f *what,
       (int)*what != CG_CONFIG_SET_PATH &&
       (int)*what != CG_CONFIG_ADD_PATH &&
       (int)*what != CG_CONFIG_HDF5_MPI_COMM) {
-    value_c = *(size_t *)value;
+    value_c = *((int *) *((size_t *)value));
```

Comment:
191 also segfaults with gcc/gfortran 7.5 at the first call to cg_configure_f

Reply:
cg_configure_f binding was not found. 
After adding the FMNAME it was working

Reply:
I missed adding it to cgns_fortrandll.def.in

Reply:
Yes HDF5 option that are of "size_t" sizes need a separate statement.
Beware that cgsize_t != size_t 
cgsize_t depends on CGNS compilation option while size_t depends on external stdlib.h .

Reply:
It looks like (void *)(*(size_t *)value works for everything except CG_CONFIG_RIND_INDEX.

Reply:
For HDF5 configure args that are size_t, I think we should require the type to be C_SIZE_T. That way we know it will be compatible with the HDF5 APIs. It seems to make sense because they are configuring HDF5 directly and bypassing CGNS.